knimin/lib/data_access_pm.py

```diff
@@ -0,0 +1,904 @@
+from knimin.lib.exceptions import *
```

Comment:
We normally list the libraries that we are importing. I guess this shows that this repo or files are not automatically being checked with flake8, could you add? What about coveralls? If another PR just adding those and then update this one is easier, we could go with that. Thanks!